contracts/script/LightClient.s.sol

```diff
@@ -145,3 +145,64 @@ contract LightClientContractUpgradeToV2Script is Script {
         return (address(implementationContract), result);
     }
 }
+
+/// @notice Upgrades the light client contract first by deploying the new implementation
+/// and then executing the upgrade via the Safe Multisig wallet using the SAFE SDK.
+contract LightClientContractUpgradeToV2PatchScript is Script {
+    string internal originalContractName = "LightClient.sol:LightClient";
+    string internal upgradeContractName = "LightClientV2.sol:LightClientV2";
+
```

Comment:
wait, i thought we are upgrading from the existing LightClientV2 to this patched version of LightClientV2? why is the origin contract being V1?

Reply:
that variable isn't even being used so I should remove

Reply:
sounds good, lgtm.